lib/spack/spack/test/cmd/env.py

```diff
@@ -721,27 +740,54 @@ def test_env_with_included_config_file():
                for x in e._get_environment_specs())
 
 
-def test_env_with_included_config_scope():
+def test_env_with_included_config_file_url(tmpdir, mutable_empty_config,
+                                           packages_file):
+    """Test configuration inclusion of a file whose path is a URL before
+       the environment is concretized."""
+
+    spack_yaml = tmpdir.join('spack.yaml')
+    with spack_yaml.open('w') as f:
+        f.write(
+            "spack:\n  include:\n    - file://{0}\n".format(packages_file))
+
+    env = ev.Environment(tmpdir.strpath)
+    ev.activate(env)
+    scopes = env.included_config_scopes()
+    assert len(scopes) == 1
+
+    cfg = spack.config.get('packages')
+    assert cfg['mpileaks']['version'] == [2.2]
+
+
+def test_env_with_included_config_missing_file(tmpdir, mutable_empty_config):
+    """Test inclusion of a missing configuration file raises FetchFailureError
+       noting missing file."""
+
+    spack_yaml = tmpdir.join('spack.yaml')
+    missing_file = tmpdir.join('packages.yaml')
+    with spack_yaml.open('w') as f:
+        f.write(
+            "spack:\n  include:\n    - {0}\n".format(missing_file.strpath))
+
+    env = ev.Environment(tmpdir.strpath)
+    with pytest.raises(FetchFailureError, match='No such file or directory'):
+        ev.activate(env)
+
+
+def test_env_with_included_config_scope(tmpdir, packages_file):
+    """Test inclusion of a package configuration file added to an existing
+       environment's configuration directory."""
     config_scope_path = os.path.join(ev.root('test'), 'config')
-    test_config = """\
-env:
-  include:
-  - %s
-  specs:
-  - mpileaks
-""" % config_scope_path
+    test_config = mpileaks_env_config(config_scope_path)
 
     _env_create('test', StringIO(test_config))
 
     e = ev.read('test')
 
     fs.mkdirp(config_scope_path)
-    with open(os.path.join(config_scope_path, 'packages.yaml'), 'w') as f:
-        f.write("""\
-packages:
-  mpileaks:
-    version: [2.2]
-""")
+    include_filename = os.path.basename(packages_file.strpath)
+    included_path = os.path.join(config_scope_path, include_filename)
+    fs.rename(packages_file.strpath, included_path)
```

Comment:
The test description says it's a test where the `package.py` is added to an existing environment's configuration directory.  The `packages_file` fixture places the file in a non-environment configuration directory; hence, the file created by the fixture is moved.

Reply:
Got it, this is minor but I think this should copy vs. move (e.g. if the fixture were not per-function, moving the resource would break other tests that use it).